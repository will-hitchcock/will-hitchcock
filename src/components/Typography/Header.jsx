import styled from 'styled-components'
import { bool, oneOf } from 'prop-types'
import { glitter, black } from '../Foundations/colors'

const Header = styled.h1`
  display: flex;
  align-items: baseline;
  justify-content: flex-start;
  color: ${({ inverted }) => (inverted ? glitter : black)};
`

Header.propTypes = {
  inverted: bool,
  size: oneOf(['small', 'medium', 'large']),
}

Header.defaultProps = {
  inverted: false,
  size: 'medium',
}

export default Header
