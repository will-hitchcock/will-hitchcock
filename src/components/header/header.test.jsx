import React from 'react'
import { shallow } from 'enzyme'
import Header from './index'

describe('<Header />', () => {
  it('should render a <Header />', () => {
    expect(shallow(<Header siteTitle="Test title!" />)).toMatchSnapshot()
  })
})
