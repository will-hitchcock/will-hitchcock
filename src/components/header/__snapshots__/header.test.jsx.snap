// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render a <Header /> 1`] = `
<styled.div>
  <styled.h1
    style={
      Object {
        "margin": 0,
      }
    }
  >
    Test title!
  </styled.h1>
</styled.div>
`;
